mandatory/3-function-output.js

```diff
@@ -3,16 +3,24 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+// This function will return a random number greater than or equal to 1 or less than 10. It will be a decimal. We don't need to pass an argument through this function.
+
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+// This function will combine 2 or more arguments passed through this function.
+// In this function, word1 will be combined with word2 in that order as word2 is inside the brackets. The function will return a new string.
+
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
Nitpick: This is fine, but reads a little funny to me. you could also do something like this
``
return `${firstWord} ${secondWord} ${thirdWord}`
``

Reply:
I hadn't thought of that. Thanks, Michael.